packages/rocketchat-message-mark-as-unread/client/actionButton.js

```diff
@@ -0,0 +1,30 @@
+Meteor.startup(function() {
```

Comment:
Places where you don't want to access the scope (this) you can use `() => {` instead of `function() {`